Plugins/Process/Collect-Prefetch.ps1

```diff
@@ -34,7 +42,9 @@
 param (
 
     [Parameter(Mandatory=$true,Position=0)]
-    [string[]]$ComputerName
+    [string[]]$ComputerName,
+    [Parameter(Mandatory=$false,Position=1)]
+    [string[]]$PrefetchName
```

Comment:
@5ynax I like this as a `String[]` for pulling multiple files in one pass, but the way you have it set up below you're expecting one file name. You can either flip this to a `String` or add a `foreach` loop inside the `if ($PrefetchName) {` check

Reply:
fixed with latest commit

Reply:
@valrkey forgot to tag